dash-renderer/package.json

```diff
@@ -33,7 +33,7 @@
     "cross-env": "^4.0.0",
     "dependency-graph": "^0.5.0",
     "json-loader": "^0.5.4",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.15",
```

Comment:
why do we even have `lodash` in our package.json? We have some deps that use it but we don't use it directly.